bots/pr/src/main/java/org/openjdk/skara/bots/pr/CheckWorkItem.java

```diff
@@ -430,6 +430,19 @@ public Collection<WorkItem> prRun(ScratchArea scratchArea) {
         var comments = prComments();
         comments = postPlaceholderForReadyComment(comments);
 
+        if (pr.headHash().hex() == null) {
+            String text = "The head hash of this pull request is missing.\n" +
+                    "Until this is resolved, this pull request cannot be processed.\n" +
+                    "Please try to use the following command to push an empty commit.\n" +
```

Comment:
```suggestion
                    "This is likely caused by a caching problem in the server and can usually be worked around by pushing another commit to the pull request branch. The commit can be empty. Example:\n" +
```
I also recommend removing all the explicit single line breaks. Single linebreaks will not be honored when formatting anyway and we don't want these sentences in separate paragraphs.